From f18eadca185f884acfe9a76204bbd0530eaf6fe7 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Sat, 14 Oct 2023 10:29:12 -0400
Subject: [PATCH] Replace ConfigParser.readfp() with ConfigParser.read_file()

The former is deprecated since Python 3.2, and was removed in Python 3.12.

https://docs.python.org/3.11/library/configparser.html#configparser.ConfigParser.readfp
---
 pypet/pypetlogging.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/pypet/pypetlogging.py
+++ b/pypet/pypetlogging.py
@@ -556,7 +556,7 @@ class LoggingManager(object):
 
         """
         parser = NoInterpolationParser()
-        parser.readfp(log_config)
+        parser.read_file(log_config)
 
         rename_func = lambda string: rename_log_file(string,
                                                      env_name=self.env_name,
